app/helpers/application_helper.rb

```diff
@@ -72,4 +72,28 @@ def content_tag_if(condition, *args, &block)
   def app_version_number
     @app_version ||= File.read(File.join(Rails.root, "VERSION"))
   end
+
+  def division_select_options(include_root: true, include_all: false, public: false)
+    divisions = public ? Division.published : current_user.accessible_divisions
```

Comment:
I think these calls should really be wrapped in a `policy_scope` call -- we don't want to be setting up any public-facing records that haven't been filtered through the policy/authorization system

Reply:
👍